### PR TITLE
Bump govmomi to 0.27.3

### PR DESCRIPTION
**Pull request by lubronzhan**
**What this PR does / why we need it**:
Our GOVC is too old. We should pick up the recent one.
Also it has bug fix for https://github.com/vmware/govmomi/pull/1154

**Which issue this PR fixes** *(optional, in `fixes #<issue number>(, fixes #<issue_number>, ...)` format, will close that issue when PR gets merged)*: fixes #https://github.com/kubernetes/cloud-provider-vsphere/issues/606

**Special notes for your reviewer**:

**Release note**:

```release-note
Bump govmomi to 0.27.3
```

### CODE REVIEW

File: pkg/common/vclib/storagepod_test.go [hunk 1]
```diff
@@ -81,7 +81,12 @@ func TestDatastoreAndClusters(t *testing.T) {
 		objs = append(objs, stores[i].Reference())
 	}
 
-	_, err = pod.MoveInto(ctx, objs)
+	task, err := pod.MoveInto(ctx, objs)
+	if err != nil {
+		t.Fatal(err)
+	}
+
+	err = task.Wait(ctx)
```
Comment (HanFa) [inline pkg/common/vclib/storagepod_test.go:89]: are we adding wait to prevent race conditions?

Comment (HanFa) [inline pkg/common/vclib/storagepod_test.go:89]: are we adding wait to prevent race conditions?

**Review by HanFa — APPROVED** [review]
LGTM

File: pkg/common/vclib/storagepod_test.go (hunk 1 above)
Comment (lubronzhan) [reply pkg/common/vclib/storagepod_test.go:89]: The new simulator will run task async. We need to wait for the task to complete before we proceed with other operations

**Pull request merged**